config/routes.rb

```diff
@@ -24,6 +26,7 @@
       end
     end
   end
+  #end
```

Comment:
If there's a card for the development piece of work we can remove these commented out things in this PR

Reply:
I left them commented out because it would mean that the live version wouldn't display anything as these routes will only display in development. So in the next story these will be uncommented out

Reply:
If the story is immediately after this one I think it's fine to just remove the lines entirely and add them back in with the next PR. But, no biggy dude. Happy as it is.